### DISCUSSION TITLE
Nimib Speaking Hours - March - Sat, March 23, 10:30 AM (CET)

### DISCUSSION THREAD

**Discussion by HugoGranstrom** [pietroppeter/nimib#234] [Nimib speaking hours]
The next Nimib Speaking Hours will take place Saturday the 23th of March at 10:30 AM (CET).
Here is a link to add it to your calendar: https://calndr.link/event/XG8wrgtA5F

**Comment by pietroppeter** [comment]
And for reference here are the notes from last time (aka today): https://github.com/pietroppeter/nimib/discussions/232#discussioncomment-8581863

**Comment by pietroppeter** [comment]
notes for today:
- meta agenda:
  - informal chat
  - will we record? will try OBS Pietro side
  - agenda
    - check notes from last meeting
    - what has happened since last time
    - did we have some pending answering/feedback tasks for users/contributors
    - what's the very non strict plan for next "spreeeeent"
  - (at least one diversion or it is not official)
  - be mindful of time
  - schedule next call
- agenda
  - check notes from last meeting [x]
  - what has happened since last time
    - Hugo started the blogpost: https://github.com/nimib-land/nblog/issues/20
    - Pietro submitted and Literate Programming and Explorable Explanations talk to PyData London conf (with NimibLand as 1/3 of topic) - still to submit to PyData Paris
    - Pietro: started the refactoring work for real
    - Pietro: a new nimislides presentation (AI a blast from the past)
      - shared it at RC (might follow up with a 5min presentation on how I make slides with code): https://github.com/pietroppeter/ai-a-blast-from-the-past
      - shared it at Lightning Talks at Py4AI (recorded, I might re-recorded again)
    - Hugo: a new nimislides presentation in preparation for university on Monday (will be public sometime in the future)
  - did we have some pending answering/feedback tasks for users/contributors
    - updated issues #236 and #101 and #237
    - discussed the merged PR #238 
    - quick release! https://github.com/pietroppeter/nimib/releases/tag/v0.3.11
  - to discuss:
    - nimib.land
      - should we migrate already?
      - we created a minimal landing page: http://nimib.land/
      - still a lot of things to check:
        - how to authorize hugo for porkbun
        - how to access nblog
        - how to have https work correctly
        - improve the landing page
    - big refactoring
      - https://github.com/nimib-land/nblog/issues/20
      - commented https://github.com/pietroppeter/nimib/blob/refactor-nb-block/sandbox/minib3/minib.nim
      - to pretty print json convert to json string with jsony and use stdlib to pretty print: https://github.com/treeform/jsony/issues/3
  - what's the very non strict plan for next "spreeeeent"
    - nimib.land?? 
    - Hugo
      - complete blogpost https://github.com/nimib-land/nblog/issues/20
      - update the logo for nimib-land
    - Pietro
      - complete refactoring
  - (at least one diversion or it is not official)
  - be mindful of time
  - schedule next call
    - April 20th same time
- diversions:
  - py4ai talks https://www.youtube.com/playlist?list=PL0RwQVm3YPu5k9iIaQUehwgh2M1DgKWaT
    - https://cheshirecat.ai/ (preview here: https://www.linkedin.com/posts/pietro-peterlongo-b7605627_py4ai-talk-by-piero-savastano-activity-7177214892075061249-EX3E?utm_source=share&utm_medium=member_desktop)
  - Python for designers: https://www.youtube.com/watch?v=hcmFn-fGe94&t=1s
  - PyMI OBS Setup https://github.com/PythonMilano/PyMI-OBS-setup (and vdo.ninja)
  - recorded again https://pietroppeter.github.io/ai-a-blast-from-the-past/

**Comment by HugoGranstrom** [comment]
Renaming the website repo to `nimib-land.github.io` seems to have solved the routing problem with nblog: https://nimib.land/nblog/

**Comment by pietroppeter** [comment]
The video recording of this Nimib Speaking Hours is being processed by the new NimibLand channel and it will be available shortly: https://www.youtube.com/watch?v=R-Qgj7yaT3o&list=PLrCsJyovXqe-eLbqMym3HfdXUdOPWFUvi
